privaterelay/settings.py

```diff
@@ -435,6 +437,273 @@ def _get_initial_middleware() -> list[str]:
+        },
+        "gb": {
+            "monthly": {
+                "id": "price_1LYCHpJNcmPzuWtRhrhSYOKB",
+                "price": "$?.??",
+            },
+            "yearly": {
+                "id": "price_1LYCIlJNcmPzuWtRQtYLA92j",
+                "price": "$?.??",
+            },
+        },
+    },
+}
+PERIODICAL_PREMIUM_PLAN_COUNTRY_LANG_MAPPING = {
+    # Austria
+    "at": {"de": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["de"]},
+    # Belgium
+    "be": {
+        "fr": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["fr"],
+        "de": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["de"],
+        "nl": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["nl"],
+    },
+    # Switzerland
+    "ch": {
+        "fr": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["chf"]["fr"],
+        "de": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["chf"]["de"],
+        "it": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["chf"]["it"],
+    },
+    # Germany
+    "de": {
+        "de": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["de"],
+    },
+    # Spain
+    "es": {
+        "es": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["es"],
+    },
+    # France
+    "fr": {
+        "fr": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["fr"],
+    },
+    # Ireland
+    "ie": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["en"],
+    },
+    # Italy
+    "it": {
+        "it": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["it"],
+    },
+    # Netherlands
+    "nl": {
+        "nl": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["nl"],
+    },
+    # Sweden
+    "se": {
+        "sv": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["sv"],
+    },
+    # Finland
+    "fi": {
+        "fi": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["fi"],
+    },
+    "us": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["en"],
+    },
+    "gb": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+    "ca": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+    "nz": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+    "my": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+    "sg": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+}
+
+PHONE_PLAN_ID_MATRIX = {
+    "usd": {
+        "en": {
+            "monthly": {
+                "id": "price_1LXUenJNcmPzuWtRw3rhjQNP",
+                "price": "$?.??",
+            },
+            "yearly": {
+                "id": "price_1LXUhcJNcmPzuWtRHUFHVk12",
+                "price": "$?.??",
+            },
+        },
+    },
+}
+PHONE_PLAN_COUNTRY_LANG_MAPPING = {
+    "us": {
+        "en": PHONE_PLAN_ID_MATRIX["usd"]["en"],
+    },
+}
+
+BUNDLE_PLAN_ID_MATRIX = {
+    "usd": {
+        "en": {
+            "yearly": {
+                "id": "price_1La3d7JNcmPzuWtRn0cg2EyH",
+                "price": "$?.??",
+            },
+        },
+    },
+}
+BUNDLE_PLAN_COUNTRY_LANG_MAPPING = {
+    "us": {
+        "en": BUNDLE_PLAN_ID_MATRIX["usd"]["en"],
+    },
```

Comment:
suggestion (non-blocking): I *think* the bundle should also be available in `ca`, because all its individual products are available in `ca` ...
```suggestion
    "us": {
        "en": BUNDLE_PLAN_ID_MATRIX["usd"]["en"],
    },
    "ca": {
        "en": BUNDLE_PLAN_ID_MATRIX["usd"]["en"],
    },
```

Reply:
Like for phones: 6b2f3e84